### PR TITLE
preventing 0 from being coerced to empty string

### PR DESCRIPTION
**Pull request by calellowitz**
@emord @kaapstorm @nickpell cc:@NoahCarnahan
http://manage.dimagi.com/default.asp?218506#1112312
when 0 was used as the period, on subsequent pageloads the 0 was coalesced to an empty string which then broke future saves. This isnt a pretty solution, and I am open to better ways but it gets the job done.

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/report-module.js [hunk 1]
```diff
@@ -159,7 +159,12 @@ hqDefine('app_manager/js/report-module.js', function () {
                     'period'
                 ];
                 for(var filterFieldsIndex = 0; filterFieldsIndex < filterFields.length; filterFieldsIndex++) {
-                    filter.selectedValue[filterFields[filterFieldsIndex]] = ko.observable(filter.selectedValue[filterFields[filterFieldsIndex]] || '');
+                    startVal = filter.selectedValue[filterFields[filterFieldsIndex]]
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/report-module.js]: Missing semicolon.

**Comment by NoahCarnahan** [comment]
I usually do something like
```js
foo = ko.observable(startVal !== undefined ? startVal : '');
```
but those statements can also be long and sorta hard to parse. I don't think your solution is so bad.
